app/helpers/casino/sessions_helper.rb

```diff
@@ -23,6 +23,10 @@ def current_user
     tgt.user
   end
 
+  def current_authenticator_context
+    CASino.config.authenticator_context_builder.call(params, request)
```

Comment:
@pencil we could have a `authenticator_context_builder_class` option that is "constantized" and set to the `authenticator_context_builder` property. The class would simply need to have a `call` method on it.

Of course, we could create a default constant class that parses config options and does something like refer to request properties, but that gets very complicated and also requires specific opinions about what the context does or consists of. We should keep it to declarative procs or classes.

Thoughts?